web/ui/dashboard/angular.json

```diff
@@ -30,7 +30,7 @@
 						"tsConfig": "tsconfig.app.json",
 						"inlineStyleLanguage": "scss",
 						"assets": ["src/favicon.ico", "src/assets"],
-						"styles": ["src/styles.scss", "node_modules/convoy-ui/scss/main.scss"],
+						"styles": ["src/styles.scss", "/Users/emmanuelaina/projects/ui-library/scss/main.scss"],
```

Comment:
@horlah can you make a pr with the changes made on the UI library? I already setup a deploy preview on netlify so we can preview the changes on the UI library before publishing

Reply:
@Oluwadaminiola fixed now.